src/argilla/client/client.py

```diff
@@ -115,7 +115,7 @@ class Argilla:
     """
 
     # Larger sizes will trigger a warning
-    _MAX_CHUNK_SIZE = 5000
+    _MAX_BATCH_SIZE = 5000
```

Comment:
@tomaarsen, FYI I believe we also need to take this into account for https://github.com/argilla-io/argilla/issues/2454

Reply:
Sounds good, shall I introduce a warning like this for `rg.load`?
https://github.com/argilla-io/argilla/blob/f5834a5408051bf1fa60d42aede6b325dc07fdbd/src/argilla/client/client.py#L338-L343

Reply:
I will not be doing this. For #2454, the server limits the maximum batch size to 500, so there is no need to have a warning when using a batch size of over 5000.